test/wrappers/ortools_test.rb

```diff
@@ -5728,7 +5728,7 @@ def test_force_start_when_there_is_a_service_with_late_tw_begin
     ]
 
     vrp[:configuration][:resolution][:duration] = 20
-    vrp[:configuration][:resolution][:first_solution_strategy] = 1
```

Comment:
Wouahou, how did this happen ?

Reply:
Yeap this is why config class is gonna be awesome :D There was another test which was broken because of a similar error.